en/docs/install-and-setup/setup/si-setup/si-cicd-k8s.md

```diff
@@ -0,0 +1,189 @@
+
+[![K8s based CI/CD for Streaming Integrator]({{base_path}}/assets/img/deploy/si-cicd-k8s.png)]({{base_path}}/assets/img/deploy/mi-cicd-k8s.png)
+
+#### Deployment Descriptor Repository
```

Comment:
Do we need to explain what these are?

Reply:
Added the explanation as the first bullet point.